libs/barista-components/chart/README.md

```diff
@@ -28,19 +28,19 @@ component about viewport changes that trigger a reflow of the `dt-chart`.
 
 ## Inputs
 
-| Name           | Type                                                                                                  | Default     | Description                                                                                                                                                       |
-| -------------- | ----------------------------------------------------------------------------------------------------- | ----------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `options`      | `Observable<DtChartOptions> | DtChartOptions | undefined`                                             | `undefined` | Sets options for the chart. `DtChartOptions` extends from `Highcharts.Options`, but removes the series property. The series property is passed as separate input. |
-| `series`       | `Observable<Highcharts.IndividualSeriesOptions[]> | Highcharts.IndividualSeriesOptions[] | undefined` | `undefined` | Sets the series of the chart. The type can either be an observable or a static array.                                                                             |
-| `loading-text` | `string`                                                                                              |             | The loading text of the loading distractor.                                                                                                                       |
+| Name           | Type                                              | Default                              | Description                                 |
+| -------------- | ------------------------------------------------- | ------------------------------------ | ------------------------------------------- |
+| `options`      | `Observable<DtChartOptions>                       | DtChartOptions                       | undefined`                                  | `undefined` | Sets options for the chart. `DtChartOptions` extends from `Highcharts.Options`, but removes the series property. The series property is passed as separate input. |
```

Comment:
I tried to commit the original file from IntellJ, Visual studio and the terminal but it gets reformatted when performing the commit (consider that I never reformatted the whole file). It does not happen only with `|` but also on `{{ }}`.
Is there any way to avoid this file formatter on the commit ?

Reply:
This should fix the issue https://www.markdownguide.org/extended-syntax/#escaping-pipe-characters-in-tables 
I thought that it was related to the ide initially but indeed the issue was the pipe inside backticks